### ISSUE TITLE
KNITRO.jl cannot be added with latest JuMP, MathOptInterface

### ISSUE THREAD

**Issue by Shuvomoy** [jump-dev/KNITRO.jl#216]
Dear All, 

If I try to install KNITRO

`add KNITRO`

it downgrades JuMP and MathoptInterface

```
  Resolving package versions...
   Installed BinaryProvider ───── v0.5.10
   Installed MutableArithmetics ─ v0.3.3
   Installed KNITRO ───────────── v0.11.1
   Installed Ipopt ────────────── v0.9.1
   Installed MosekTools ───────── v0.11.0
   Installed JuMP ─────────────── v0.22.3
   Installed MathOptInterface ─── v0.10.9
    Updating `C:\Users\shuvo\.julia\environments\v1.6\Project.toml`
  [b6b21f68] ↓ Ipopt v1.0.1 ⇒ v0.9.1
  [4076af6c] ↓ JuMP v0.23.0 ⇒ v0.22.3
  [67920dd8] + KNITRO v0.11.1
  [1ec41992] ↓ MosekTools v0.12.0 ⇒ v0.11.0
    Updating `C:\Users\shuvo\.julia\environments\v1.6\Manifest.toml`
  [b99e7846] + BinaryProvider v0.5.10
  [b6b21f68] ↓ Ipopt v1.0.1 ⇒ v0.9.1
  [4076af6c] ↓ JuMP v0.23.0 ⇒ v0.22.3
  [67920dd8] + KNITRO v0.11.1
  [b8f27783] ↓ MathOptInterface v1.1.0 ⇒ v0.10.9
```

**Comment by frapac** [comment]
This is expected, as KNITRO.jl supports MOI 1.0 only on `master`. A new release is expected soon.

**Comment by Shuvomoy** [comment]
Okay thanks @frapac !

**Comment by frapac** [comment]
Hi,
Knitro.jl 0.12.0 has been released, bringing support for MOI 1.0. 
Closing this issue now, but feel free to reopen if you encounter any issue with the new version.
